### PR TITLE
Antag loadout refactor, also deathsquads I guess

### PR DESCRIPTION
**Pull request by JohnWildkins**
- all antags that previously had massive equip chains now use the outfit datum system
- commandos + deathsquad (aka Syndicate Commandos and NT Asset Protection) have been moved out of nested alert menu hell to the new ERT system as admin-spawned ERTs, they do not appear normally
- all outfit datums will now take into account backpack preferences
- made the ERT job announcement / details more obvious

to-do:
- [x] find a better way to check for backpack preferences
- [x] test everything
- [x] strip out the last vestiges of the old strike team system
- [x] did i say test everything

### CODE REVIEW

**Comment by JohnWildkins** [comment]
!wip

**Comment by JohnWildkins** [comment]
!review

File: code/game/gamemodes/heist/heist.dm [hunk 1]
```diff
@@ -6,8 +6,8 @@ VOX HEIST ROUNDTYPE
 /datum/game_mode/heist
 	name = "heist"
 	config_tag = "heist"
-	required_players = 15
-	required_enemies = 4
+	required_players = 1
+	required_enemies = 1
```
Comment (fernerr) [inline code/game/gamemodes/heist/heist.dm]: You may have forgotten something

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 2]
```diff
@@ -11,8 +11,8 @@ var/list/nuke_disks = list()
 		and many seek to undermine them using illegal ways. Their crown jewel research stations are not safe from those \
 		malicious activities."
 	config_tag = "mercenary"
-	required_players = 15
-	required_enemies = 4
+	required_players = 1
+	required_enemies = 1
```
Comment (fernerr) [inline code/game/gamemodes/nuclear/nuclear.dm]: Ditto

File: code/game/gamemodes/heist/heist.dm (hunk 1 above)
Comment (Geevies) [reply code/game/gamemodes/heist/heist.dm]: working as intended

**Pull request merged**